birdvoxpaint/birdvoxpaint.py

```diff
@@ -1,76 +1,110 @@
+from functools import partial
+
 import librosa
-from librosa.util.exceptions.ParameterError
+import librosa.display
 import numpy as np
 
+import tqdm
+import joblib
+
+from . import util
+from .indices import *
+from .display import *
+
+MAX_JOBS = 40
+
+def transform(filename=None, y=None, sr=None,
+              frame_length=2048, hop_length=512,
+
+              n_mels=256, fmin=None, fmax=None,
+              time_constant=0.4, use_pcen=True, use_mels=True,
+
+              indices=[average_energy],
+              segment_duration=10,
+              verbose=True, n_jobs=None, return_plot=False):
+    '''Convert audio into a false color spectrogram.
```

Comment:
This description is a bit vague. There is no false-color mechanic in this function. I would prefer `Extract spectrotemporal acoustic indices in a monophonic audio file`

Reply:
although maybe i am the one who originally wrote this, in case i am effectively dunking at myself?

Reply:
I literally crammed that in 15 seconds before pushing because I felt bad abt not having a docstring 😂 but yeah that sounds better